docs/standards/generic-standards/lsp17-contract-extension.md

```diff
@@ -0,0 +1,90 @@
+---
+sidebar_label: 'LSP17 - Contract Extension'
+sidebar_position: 4
+---
+
+# LSP17 - Contract Extension
+
+:::info Standard Document
+
+[LSP17 - Contract Extension](https://github.com/lukso-network/LIPs/blob/main/LSPs/LSP-17-ContractExtension.md)
+
+:::
+
+## Introduction
+
+Once a contract is deployed on the blockchain, it is not possible to modify the contract to add new native functions or change the behavior of existing ones. This can be a limitation for smart contracts, particularly those that may need to adapt to new use cases and standards over time.
+
+What is required is a method to extend the functionalities of a smart contract even after it has been deployed, allowing it to continue to support new capabilities over time.
+
+A possible solution to this problem is to establish a system of extensions that can be added to a smart contract, enabling it to **acquire new functionalities** without the need for redeployment.
+
+## What does this standard represent ?
+
+This standard defines a mechanism for extending a contract to support new functions through the use of **extensions**.
+
+![Normal contract Vs contract implementing LSP17](/img/standards/lsp17/TwoContracts.jpeg)
+
+The **_smart contract A_** implements 4 functions natively. Once this smart contract is deployed, no new functions can be added or changes made to the existing ones.
+
+In contrast, the **_smart contract B_**, which supports the **[LSP17 standard](#)**, does not implement any native functions but rather relies on extensions for the functionality that is needed and can add more extensions in the future.
+
+Thus, Smart contract B has the ability **to support an unlimited number of functionalities** and can add new functions in the future if desired.
```

Comment:
i tihnk you should write it the same way you wrote:

```
**_smart contract A_**
```

at line 28, for consistency.

Reply:
or with a small s at the beginning.